packages/web-components/fast-foundation/src/design-token/design-token.ts

```diff
@@ -143,7 +143,7 @@ class DesignTokenImpl<T extends { createCSS?(): string }> extends CSSDirective
         return [...this._appliedTo];
     }
 
-    public static from<T>(
+    public static from<T extends { createCSS?(): string }>(
```

Comment:
Would it be good to create a couple of local types to consolidate some of these type concerns. Namely:

```ts
type DesignTokenType = string | number | boolean | BigInteger | null | Array<any> | symbol | {};

type CSSDesignTokenType = DesignTokenType |
        ({
              createCSS?(): string;
         } & Record<PropertyKey, any>);
```

Then you could do something like this everywhere:

```ts
public static from<T extends CSSDesignTokenType> ...
```